wdl/transforms/new-base/src/test/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluatorsSpec.scala

```diff
@@ -0,0 +1,26 @@
+package wdl.transforms.base.linking.expression.values
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class EngineFunctionEvaluatorsSpec extends AnyFlatSpec with Matchers{
+  behavior of "read_lines"
+  it should "return an empty array from an empty file" in {
+    EngineFunctionEvaluators.split_lines("") shouldBe(List.empty)
+  }
+  it should "return a one element array with a one line file" in {
+    EngineFunctionEvaluators.split_lines("one line") shouldBe(List("one line"))
+  }
+  it should "return a one element array with a one line file ending with a new line character" in {
+    EngineFunctionEvaluators.split_lines("one line\n") shouldBe(List("one line"))
+  }
+  it should "return a one element array with a single new line character" in {
+    EngineFunctionEvaluators.split_lines("\n") shouldBe(List(""))
+  }
+  it should "return a three element array with three new line characters" in {
```

Comment:
The description of this test case is not what the method actually does.

Reply:
Is this referencing using the word 'array' as opposed to the word 'List'.